TEST-PLAN.md

```diff
@@ -0,0 +1,123 @@
+# Judge.me Reviews Test Plan
+# Overview
+This test automation suite validates the core functionality of the Judge.me reviews system. 
+The tests are implemented using Cypress and follow the Page Object Model pattern for maintainability and reusability.
+
+## **Setup and Installation**
+
+    ## Prerequisites
+    1. Node.js (v12 or higher)
+    2. npm (v6 or higher)
+
+## Setup
+    - Clone repository:
+        git clone  https://github.com/kveenaram/judge-me-tests
+        cd judge-me-tests
+
+## Install dependencies:
+        npm install
+
+## Run Cypress tests:
+        npx cypress open
```

Comment:
Why `npx` here and not the scripts you added into the package.json

Reply:
Good question about using 'npx cypress open' directly vs package.json scripts. While we could define this in package.json scripts, I chose to document the direct npx command here in the TEST-PLAN.md for a few reasons:

Transparency - It shows explicitly what command is being run without requiring users to check package.json
Learning purposes - For new users, seeing the actual npx command helps understand what's happening under the hood
Flexibility - Users can run this command from any directory without needing to be at the root where package.json is located

However, you make a good point - we do have these commands in package.json scripts for convenience. I can update the documentation to show both approaches
# Run Cypress tests:
npx cypress open    # Direct command
# OR
npm run cypress    # Using package.json script